docs/wiki/framework/rearm-framework.md

```diff
@@ -39,16 +39,37 @@ class CfgVehicles {
 </div>
 
 ### 1.2 Setting vehicle as a supply
+
+
+#### 1.2.1 Setting up in the Config
+
 A vehicle will be set as a supply vehicle based on the config `ace_rearm_defaultSupply`
 ```cpp
 class MyTruck: Car_F {
     ace_rearm_defaultSupply = 1200;
 };
 ```
-<div class="panel callout">
-    <h5>Note:</h5>
-    <p>Mission makers can also use `this setVariable ["ace_rearm_isSupplyVehicle", true]`</p>
-</div>
+
+#### 1.2.2 Make an object into a rearm source
+*Added in ACE3 3.12.5*
+
+`ace_rearm_fnc_makeSource`
+
+   | Arguments | Type | Optional (default value)
+---| --------- | ---- | ------------------------
+0  | Rearm Source | Object | Required
+1  | Amount | Number | Optional (default: `0`)
+1  | Add instead set | Bool | Optional (default: false)
```

Comment:
`Add instead of set` would be a lot more readable to me.

Reply:
My initial thought was to keep it short as not to mess with the layout, cause I don't know how to see the exact result for the tables locally. But looking at how other tables handle overflow I changed it to the more verbose `Add amount instead of setting it` and just hope that it will be *fine*.

Reply:
> Add (`false`) or set (`true`) amount

Would that be an acceptable compromise?

Reply:
Fine by me

Reply:
Done